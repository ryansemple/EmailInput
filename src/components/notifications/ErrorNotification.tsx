import React from "react";
import Notification from "./Notification";

interface ErrorNotificationProps {
	text: string
}

const ErrorNotification = (props: ErrorNotificationProps) => (
	<Notification 
		text={props.text}
		className="Notifications_Notification-Error"
	/>
)

export default ErrorNotification;