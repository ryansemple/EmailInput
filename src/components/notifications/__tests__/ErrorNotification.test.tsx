import React from "react";
import { render } from "@testing-library/react";
import ErrorNotification from "../ErrorNotification";

test("Renders ErrorNotification component correctly.", () => {
	const { container } = render(
		<ErrorNotification 
			text="Error"
		/>
	);
	expect(container).toMatchSnapshot();
});