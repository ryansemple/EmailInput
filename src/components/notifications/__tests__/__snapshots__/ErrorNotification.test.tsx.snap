// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ErrorNotification component correctly. 1`] = `
<div>
  <div
    class="Notifications_Notification Notifications_Notification-Error"
    role="alert"
  >
    <p>
      Error
    </p>
  </div>
</div>
`;
